src/plugins/content_management/server/core/core.ts

```diff
@@ -53,4 +61,15 @@ export class Core {
       },
     };
   }
+
+  private setupEventStream() {
+    // TODO: This should be cleaned up and support added for all CRUD events.
+    this.eventBus.on('deleteItemSuccess', (event) => {
+      this.ctx.eventStream.addEvent({
+        // TODO: add "subject" field to event
+        predicate: ['delete'],
+        object: [event.contentTypeId, (event as any).contentId],
+      });
+    });
```

Comment:
This will be a separate work item.

Reply:
I'm thinking maybe Event Bus should also have the *subject-predicate-object* concept. I.e. the Event Bus events should contain precomputed *subject*, say `['user', 'xyz']` or `['api-key', '123']`. And the `.contentId` of the *object* should also always be available, where possible.

Reply:
Tracking this here: https://github.com/elastic/kibana/issues/153258